src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -2729,7 +2729,8 @@ MavlinkReceiver::handle_message_hil_state_quaternion(mavlink_message_t *msg)
 		hil_local_pos.vxy_max = INFINITY;
 		hil_local_pos.vz_max = INFINITY;
 		hil_local_pos.hagl_min = INFINITY;
-		hil_local_pos.hagl_max = INFINITY;
+		hil_local_pos.hagl_max_z = INFINITY;
+		hil_local_pos.hagl_max_xy = INFINITY;
```

Comment:
"set to 0 when limiting not required" ? Is that actually true or do we really need to set them to `INFINITY`?

Reply:
IMO infinity is the better default for the max height etc